pages/linux/setfont.md

```diff
@@ -0,0 +1,20 @@
+# setfont
+
+> Set the terminal font.
+> More information: <https://manned.org/setfont>.
+
+- Change the terminal font:
+
+`setfont {{font}}.gz`
+
+- Change the terminal font by specifying the full filepath:
+
+`setfont /usr/share/kbd/{{consolefonts|unimaps|consoletrans}}/{{font}}.gz`
+
+- Double font size:
+
+`setfont {{[-d|--double]}}`
```

Comment:
Are the square brackets correct or a typo?
```suggestion
`setfont {{-d|--double}}`
```

Reply:
It is correct. They're part of the new syntax we added to the client specifications. The client is allowed to interpret it to show either of them or both which makes the user-visible page less cluttered.